sound/soc/sof/intel/hda-dsp.c

```diff
@@ -595,11 +595,8 @@ static int hda_suspend(struct snd_sof_dev *sdev, bool runtime_suspend)
 		return ret;
 	}
 
-#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
 	/* display codec can powered off after link reset */
-	if (HDA_IDISP_CODEC(bus->codec_mask))
-		hda_codec_i915_display_power(sdev, false);
-#endif
+	hda_codec_i915_display_power(sdev, false);
```

Comment:
this looks odd, you remove both the dependency on HDA and the test on codec_mask. What is the intent?

Reply:
@plbossart These are now moved inside hda_codec_i915_display_power().

Reply:
ok then, thanks!